sphinx/conf.py

```diff
@@ -394,5 +394,7 @@ def getValue(value):
     r'https://www.excelitas.com/*', # 403 Client Error: Forbidden
     r'https://www.sas.upenn.edu/*', # certificate issuer cannot be verified
     r'https://www.microsoft.com/en-us/download/details.aspx?id=48145', # 403 Client Error: Forbidden
-    r'https://www.agilent.com/home' # 403 Client Error: Forbidden
+    r'https://www.agilent.com/home', # 403 Client Error: Forbidden
+    r'https://www.mathworks.com/*', # 403 Client Error: Forbidden
```

Comment:
https://merge-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-linkcheck/288/ still shows errors for 6 `https://mathworks.com/...` links. Probably `r'https://mathworks.com/*'` needs to be added separately, or all links need to be unified to `https://www.mathworks.com/...` instead of a mix of that and `https://mathworks.com/...`.